source/val/validate_layout.cpp

```diff
@@ -46,6 +47,19 @@ spv_result_t ModuleScopedInstructions(ValidationState_t& _,
                  << "Non-semantic OpExtInst must not appear before types "
                  << "section";
         }
+      } else if (spvExtInstIsDebugInfo(inst->ext_inst_type())) {
+        // Debug info extinst opcodes other than DebugScope, DebugNoScope,
+        // DebugDeclare, DebugValue must be placed between section 9 (types,
+        // constants, global variables) and section 10 (function declarations).
+        if (_.current_layout_section() < kLayoutTypes ||
+            _.current_layout_section() >= kLayoutFunctionDeclarations) {
+          return _.diag(SPV_ERROR_INVALID_LAYOUT, inst)
+                 << "OpenCL.DebugInfo.100 instructions other than "
```

Comment:
Nit:  This applies to both OpenCL.DebugInfo.100 and DebugInfo

Reply:
I changed the wording "Debug info extension instructions other than .. "